src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -9,19 +9,20 @@
     <PackageTargetRuntime>$(MinOSForArch)-$(PackagePlatform)</PackageTargetRuntime>
   </PropertyGroup>
   <ItemGroup>
-    <ArchitectureSpecificNativeFile Include="$(BinDir)clretwrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)coreclr.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)dbgshim.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordaccore.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)clretwrc.dll" />
```

Comment:
Can you please make similar change in closed build since Arm32 package is published from there?

Also, please work with @wtgodbe to ensure that these get published to myget.